feedback/static/feedback.css

```diff
@@ -338,6 +338,11 @@ body {
 		justify-content: flex-start;
 	}
 }
+.feedback-response-panel > .panel-body > .conversation-tag-panel > .colortag:not(:hover):not(.colortag-active) {
+	background-color: white !important;
+	color: white;
+	border: gray 1px solid;
```

Comment:
I would also suggest making the border color lightgray rather than gray, so it would be a bit more calm and lower contrast.